plugins/eslint-config/index.js

```diff
@@ -82,6 +82,12 @@ module.exports = {
   overrides: [
     {
       files: ['**/*.mocha.js'],
+      settings: {
+        jsdoc: {
+          mode: 'closure',
```

Comment:
I'm not sure whether this causes any unwanted side effects. 
It is needed for @template tag support

Reply:
I could also set this config for all the files. This change here only changes the mode for mocha files.